receiver/k8sclusterreceiver/receiver_test.go

```diff
@@ -254,6 +291,10 @@ func TestReceiverWithMetadata(t *testing.T) {
 	require.NoError(t, r.Shutdown(ctx))
 }
 
+func createComponentWithID(id string) component.ID {
+	return component.MustNewID(id)
+}
+
```

Comment:
Can you just use `component.MustNewID(id)` ?

Reply:
done